src/main/java/com/lpvs/entity/LPVSPullRequest.java

```diff
@@ -32,7 +32,7 @@ public class LPVSPullRequest implements Serializable {
     @Column(name = "scan_date")
     private Date date;
 
-    @Column(name = "user")
+    @Column(name = "user_string")
```

Comment:
What is the meaning of "user_string"? Change to a meaningful name.

Reply:
The meaning is that name "user" is a keyword and that causes an error when executing schema-h2.sql file.

Reply:
I will look into how this can be fixed by other means.

Reply:
Just rename the field using more meaningful name.